lib/rules/jsx-no-target-blank.js

```diff
@@ -38,19 +41,40 @@ module.exports = {
       category: 'Best Practices',
       recommended: true
     },
-    schema: []
+    schema: [{
+      type: 'object',
+      properties: {
+        links: {
+          type: 'boolean',
+          default: true
+        },
+        forms: {
+          type: 'boolean',
+          default: false
+        }
+      },
+      additionalProperties: false
+    }]
   },
 
   create: function(context) {
     return {
       JSXAttribute: function(node) {
-        if (node.parent.name.name !== 'a') {
+        const config = Object.assign({
+          links: true,
+          forms: false
+        }, context.options[0]);
+
+        if (
+          (config.links ? node.parent.name.name !== 'a' : true) &&
```

Comment:
`!config.links || node.parent.name.name !== 'a'`?

Reply:
Yeah I was thinking about it, do you think it is more readable?

Reply:
hm, i'm not really sure. i'm not a fan of ternaries that select booleans, but as-is might be more readable in that case.